jaggeryapps/admin/serviceproviders/editsp.jag

```diff
@@ -1136,8 +1141,11 @@ $("#custom-app-dropdown .dropdown-menu li a").click(function () {
             $("#security-accordion").show();
             $("#cladimconfig-header").remove();
             $("#inbound-auth-header").hide();
+            //set app type description
+            $("#app-type-desc-span").text("Application should be able to generate standard SSO requests and process responses.");
+            $("#app-type-desc-url").attr('href', 'https://docs.wso2.com/display/IdentityCloud/Configuring+Agent-based+SSO');
```

Comment:
Better to move these documentation links to some conf file and refer from that here.  So if the documentation links got changed it's easy to locate them